browser/ipfs/test/BUILD.gn

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/build/config.gni")
+import("//brave/components/ipfs/buildflags/buildflags.gni")
+import("//testing/test.gni")
+
+source_set("brave_browser_ipfs_unittests") {
```

Comment:
nit: just unit_tests & browser_tests seems better to me, as we'll use its path when we include it, ex: `//brave/browser/ipfs/test:brave_browser_ipfs_unittests`, brave_browser_ipfs prefix here seems redundant to me.